src/modules/lua_mtev.c

```diff
@@ -3545,6 +3761,8 @@ static const luaL_Reg mtevlib[] = {
   { "spawn", nl_spawn },
   { "thread_self", nl_thread_self },
   { "eventer_loop_concurrency", nl_eventer_loop_concurrency },
+  { "interstate_set", nl_interstate_set},
+  { "interstate_get", nl_interstate_get},
```

Comment:
interstate sounds a bit odd... can we change this to "shared_set" and "shared_get" ?